docs/main/android/index.md

```diff
@@ -58,6 +58,8 @@ npx cap run android
 
 The command will prompt you to select a target. [Learn more about `run`](/docs/cli/commands/run).
 
+> An Android Virtual Device (AVD) is required to use the `run` command. See the [documentation here for creating and installing AVD modules](https://developer.android.com/studio/run/managing-avds).
```

Comment:
An AVD is not required to use the run command, you can also run on a real device even if you don't have any AVD installed.
Also native-run is supposed to create one AVD if you don't have any and also there are no devices connected.

What you need is at least on emulator "system image" downloaded so native-run can create an emulator and that's usually what's missing when you run into the ERR_UNSUITABLE_API_INSTALLATION.

We have it documented on the native-run wiki

https://github.com/ionic-team/native-run/wiki/Android-Errors#err_unsuitable_api_installation

Reply:
Ah, thanks! Updated to clarify real device or emulator system image required. The native-run wiki instructions to pass `--sdk-info` requires running the native-run command, which in this case is being called by the capacitor command, so I think including the requirement here in the docs is still a good callout.